skimage/feature/tests/test_corner.py

```diff
@@ -31,22 +32,32 @@ def im3d():
 def test_structure_tensor():
     square = np.zeros((5, 5))
     square[2, 2] = 1
-    Axx, Axy, Ayy = structure_tensor(square, sigma=0.1)
```

Comment:
maybe you could keep this part in the test with an explicit `order='xy'`? We're deprecating it but it's still supported at the moment so we might as well keep the test.

Reply:
never mind, the function you added below does just this :-). I let you resolve the convo.